File: 프로그래머스/0/181847. 0 떼기/0 떼기.java
class Solution {
    public String solution(String n_str) {
        int count = 0;
        for(int i = 0; i < n_str.length(); i++){
            if(n_str.charAt(i) != '0'){
                count = i;
                break;
            }
        }
        char [] ch = new char [n_str.length()-count];
        int c = 0;
        for(int i = 0; i < ch.length; i++){
            ch[i] = n_str.charAt(count++);
        }
        String answer = new String(ch);
        return answer;
            
    }
}

File: 프로그래머스/0/181847. 0 떼기/0 떼기.cs
using System;

public class Solution {
    public string solution(string n_str) {
        int count = 0;
        for (int i = 0; i < n_str.Length; i++) {
            if (n_str[i] != '0') {
                count = i;
                break;
            }
        }
        
        char[] ch = new char[n_str.Length - count];
        int c = 0;
        for (int i = 0; i < ch.Length; i++) {
            ch[i] = n_str[count++];
        }
        
        string answer = new string(ch);
        return answer;
    }
}
